ContextualPopupDecorator/tests/ContextualPopupDecorator-specs.js

```diff
@@ -79,7 +134,64 @@ describe('ContextualPopupDecorator Specs', () => {
 		expect(handleClose).not.toHaveBeenCalled();
 	});
 
-	test('should have "below right" className when direction is set to "below right"', () => {
+	test('should have \'below\' className when direction is set to \'below\'', () => {
+		const handleClose = jest.fn();
```

Comment:
is `handleClose` needed if we don't use it in the test case?
same for the other tests

Reply:
No, I must have left it by mistake. Thank you for pointing it out!